.build/external-documentation/cli-getting-started.quickstart.yml

```diff
@@ -4,6 +4,11 @@ metadata:
   name: docs-cli-getting-started
   annotations:
     order: 1
+  labels:
+    service-category/application-services: "true"
+    product/rhosak: "true"
+    product/rhosr: "true"        
+    product/rhoc: "true"  
```

Comment:
This doc should probably have the "technology: cli" label too?

Reply:
added thanks @bredamc